third_party/proton/proton/profile.py

```diff
@@ -3,6 +3,7 @@
 import os
 
 from triton._C.libproton import proton as libproton
+from triton.language import core as tl
```

Comment:
Why do we need `tl` here?

Reply:
Updated. Artifact from before we moved the record function into language.py